keps/sig-node/1287-in-place-update-pod-resources/README.md

```diff
@@ -984,18 +997,15 @@ later accepted. This metric primarily exists because if a deferred resize is acc
 opposed to being triggered by an event such as another pod being deleted or sized down), it indicates an issue in the Kubelet's logic for handling deferred resizes that we should fix.
 
 Labels:
-  - `accepted_reason` - whether the resize was accepted through the timed retry or due to another pod event. Possible values: `periodic_retry`, `event_based`.
-  - `namespace` - the namespace of the pod.
+  - `retry_trigger` - whether the resize was accepted through the timed retry or due to another pod event. Possible values: `periodic_retry`, `pod_resized`, `pod_updated`, `pods_added`, `pods_removed`.
```

Comment:
ditto my above comments about implementation history to explain the evolution of the labels in the `kubelet_pod_deferred_resize_accepted_total` metric

Reply:
same response as above